docs/docs_source/wopi/files/RefreshLock.rst

```diff
@@ -14,17 +14,35 @@ RefreshLock
     The refreshed lock must expire automatically after 30 minutes unless it is modified by a subsequent WOPI
     operation, such as :ref:`Unlock` or :ref:`RefreshLock`.
 
+    ..  include:: /fragments/priorlock.rst
+
+    ..  include:: /fragments/lock409.rst
+
+    ..  include:: /fragments/no_lock_id.rst
+
+    ..  note::
+        Since there is no GetLock operation defined in WOPI, Office Online will sometimes make a RefreshLock call to
+        the host with an invalid lock value, expecting the host to respond with a :http:statuscode:`409` and an
+        **X-WOPI-Lock** response header containing the value of the current lock on the file. This technique is used
+        both to retrieve the contents of the current lock as well as test whether a file is locked at all.
+
+
     ..  include:: /fragments/common_params.rst
 
     :reqheader X-WOPI-Override:
         The **string** ``REFRESH_LOCK``. Required.
     :reqheader X-WOPI-Lock:
-        A **string** provided by Office Online that the host must use to identify the lock on
+        A **string** provided by Office Online that the host must use to identify the existing lock on
         the file. The maximum length of a lock ID is 256 characters.
```

Comment:
I think Chris pointed this out something similar to this in one of the meetings. Do we actually want to say this, as we don't know this will be true forever. Instead maybe something like the lockId requires a minimum 256 character storage.